### PR TITLE
Added Fetch and getUTC help descriptions

### PR DESCRIPTION
**Pull request by Zachdehooge**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -40,13 +40,13 @@ async def send_pages(self):
...
 
-@client.command(name="getUTC")
+@client.command(name="getUTC", help="Get the current UTC time.")
 async def getUTC(ctx) -> None:
     utc_time = await getUTCTime()
     await ctx.send(utc_time.strftime("%Y-%m-%d %H:%M:%S"))
 
 
-@client.command(name="fetch")
+@client.command(name="fetch", help="Fetch the latest Nadocast images. \n Usage: !fetch <param>\nAllowed params: sig, life, tor, wind, hail")
```
Comment (Arch881010) [inline main.py:49]: Current code uses $ for the prefix.
$fetch sig tor
$fetch tor are both proper 

$fetch life hasn't been implemented (yet)

**Review by Arch881010 — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (Zachdehooge) [reply main.py:49]: Ah my brain stopped working, yeah it should be $ instead

File: main.py [hunk 2]
```diff
@@ -40,13 +40,13 @@ async def send_pages(self):
 client.help_command = MyHelpCommand()
 
 
-@client.command(name="getUTC")
+@client.command(name="getUTC", help="Get the current UTC time.")
```
[the remark this answers is not in the document]
Comment (Zachdehooge) [reply main.py:43]: Done!

File: main.py [hunk 3]
```diff
@@ -40,13 +40,13 @@ async def send_pages(self):
...
 
-@client.command(name="getUTC")
+@client.command(name="getUTC", help="Gets the current UTC time.")
 async def getUTC(ctx) -> None:
     utc_time = await getUTCTime()
     await ctx.send(utc_time.strftime("%Y-%m-%d %H:%M:%S"))
 
 
-@client.command(name="fetch")
+@client.command(name="fetch", help="Fetch the latest Nadocast images. \n Usage: $fetch <param> \n Allowed params: sig, tor, wind, hail")
```
Comment (Arch881010) [inline main.py:49]: No clue if the user would assume only one parameter is expected. Maybe the word should be re-worded? 
 ```suggestion
@client.command(name="fetch", help="Fetch the latest Nadocast images. \n Usage: $fetch <params> \n Allowed params: sig, tor, wind, hail\n Examples: `$fetch tor`, `$fetch sig tor`")
```

**Review by Arch881010 — APPROVED** [review]
Overall LGTM and can be pushed to main. Please see possible suggestion, though.

File: main.py (hunk 3 above)
Comment (Zachdehooge) [reply main.py]: Oh cool didn't know you can just commit the suggestion instead of doing it manually from my end lol

Comment (Arch881010) [reply main.py]: Yeah I found out how to use suggestions lol

**Comment by Zachdehooge** [comment]
Anything else before I merge you can think of?

**Comment by Arch881010** [comment]
> Anything else before I merge you can think of?

Didn't see this, and nope. Nice work!

**Pull request merged**